core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -286,7 +288,7 @@ class ConsumerBounceTest extends AbstractConsumerTest with Logging {
 
     servers.foreach(server => killBroker(server.config.brokerId))
     val closeTimeout = 2000
-    val future1 = submitCloseAndValidate(consumer1, closeTimeout, Some(closeTimeout), Some(closeTimeout))
+    val future1 = submitCloseAndValidate(consumer1, closeTimeout, None, Some(closeTimeout))
```

Comment:
This is needed since it previously is based on a transient state that there are un-committed offsets yet to be sent upon closing, with this PR the likelihood of it is much smaller causing it to be flaky.